ci_build_images/msan.Dockerfile

```diff
@@ -116,4 +116,4 @@ RUN apt-get source cracklib2 \
     && chmod -R a+x $MSAN_LIBDIR/bin/*
 
 ENV CFLAGS="-fno-omit-frame-pointer -O2 -g -fsanitize=memory"
-ENV CXXFLAGS="$CFLAGS"
+ENV CXXFLAGS="$CFLAGS"
```

Comment:
?

Reply:
not sure what's different here and why it appears as a change

Reply:
me neither :)

Reply:
Ah, maybe a extra line at the end (some IDE automatically adds them)...

Reply:
yes, but as far as I checked there wasn't a new line before and there isn't now. But probably something related to newlines